crates/swc/tests/tsc-references/es6/destructuring/destructuringParameterDeclaration1ES5/input.ts/es2015.2.minified/output.js

```diff
@@ -1,4 +1,4 @@
-var Foo, Foo1;
+var Foo, Foo;
```

Comment:
Again, not sure why this is happening, but it seems ok and it's better than before. We should try to eliminate this sometime though.